easybuild/easyconfigs/f/fontconfig/fontconfig-2.13.96-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'fontconfig'
+version = '2.13.96'
```

Comment:
There is a `2.14.00` available. All the `2.13.9X` are described as `2.14 RC6`. Should we update?

Reply:
good point, I'll update

Reply:
Can you delete the `2.13.96` from this PR?